client/src/views/admin/index.vue

```diff
@@ -30,7 +30,7 @@
         </b-container>
       </b-card>
     </b-container>
-    <router-link to="/admin/userList">User</router-link>
+    <button><router-link to="/admin/userList">User</router-link></button>
     <router-view></router-view>
```

Comment:
pls remove and add router here:
https://github.com/joelmjohn/foreignmentor-uk/blob/e08439be68c54aee885128a03f26337a63ab667b/client/src/App.vue#L9C1-L10